src/OSPSuite.R/Services/SimulationRunner.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Data;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Core.Extensions;
+using OSPSuite.Core.Maths.Random;
+
+namespace OSPSuite.R.Services
+{
+   public interface ISimulationRunner
+   {
+      Task<SimulationResults> RunSimulationAsync(IModelCoreSimulation simulation, SimulationRunOptions simulationRunOptions = null);
+      SimulationResults RunSimulation(IModelCoreSimulation simulation, SimulationRunOptions simulationRunOptions = null);
+   }
+
+   public class SimulationRunner : ISimulationRunner
```

Comment:
This is dummy implementation for now. It is required to get going until SimModel works